cirq/ops/pauli_string.py

```diff
@@ -395,7 +398,9 @@ def _expectation_from_wavefunction_no_validation(
     def expectation_from_density_matrix(self,
                                         state: np.ndarray,
                                         qubit_map: Mapping[raw_types.Qid, int],
-                                        atol: float = 1e-7) -> float:
+                                        atol: float = 1e-7,
```

Comment:
While you're at it, force keywords starting with `atol`.

Add `check_preconditions` to the `Args:` list.